docs/source/docking_multiple_ligands.rst

```diff
@@ -41,7 +45,7 @@ The output PDBQT ``5x72_ligand_p59.pdbqt`` and ``5x72_ligand_p69.pdbqt`` can be
 3. (Optional) Generating affinity maps for AutoDock FF
 ------------------------------------------------------
 
-As well as for the docking with a fully rigid receptor, we need to generate a GPF file to precalculate the affinity map for each atom types. However, instead of using the full receptor, affinity maps will be calculated only for the rigid part of the receptor (``5x72_receptor.pdbqt``). The Python script ``prepare_gpf.py`` is available here: ``<autodock-vina_directory>/example/autodock_scripts``.
+As well as for the docking with a fully rigid receptor, we need to generate a GPF file to precalculate the affinity map for each atom types. However, instead of using the full receptor, affinity maps will be calculated only for the rigid part of the receptor (``5x72_receptor.pdbqt``).
```

Comment:
precalculate the affinity map for each atom types -> precalculate the affinity maps for each atom type